### DISCUSSION TITLE
Sticky Header Not Showing Background Image

### DISCUSSION THREAD

**Discussion by aroundi** [templaza/astroid-framework#306] [General]
Hello,

I started experimenting with Astroid Framework using template One. Ver 2.6.4. When the Sticky Header is activated the bg image set for the header section disappears. I tested setting the column and element bg image respectively; the image does not span full width which does not suit the site design and I got the same result.

I'm new to Astroid so I'm not sure if I've missed something in the configuration. Any ideas are appreciated.

Thank you,

mark

**Comment by Chacapamac** [comment]
I @aroundi 
I’m sure you will be impress by the Framework. I will try to help

> When the Sticky Header is activated the bg image set for the header section disappears.
Yes, if I understand well, this is normal behavior, look at my example here —> http://joom4-astro.graf-test.ca
— When the Sticky Header is out, normally the “Header” position (and background) is not visible, only when you are really close to the top of the page. Only there, the sticky go away when the “Header” position is in view.

The Sticky is a div name #astroid-sticky-header in a fixed position that is out of the normal flow (Layouts positions) of the page. If you want to have a background image to the  “Sticky” for now you will have to set it in custom css using something like 
—> #astroid-sticky-header {background-image: url(yourimagepath);
You can change colors of the Sticky in **Astroid Layout Manager —> Colors**
![Screen Shot 2023-03-15 at 8 31 01 AM](https://user-images.githubusercontent.com/1907276/225309614-ee3c3d81-ed79-4c80-9acc-047e4ec60078.jpg)

> I tested setting the column and element bg image respectively; the image does not span full width which does not suit the site design and I got the same result.
Can you give me a screenshot for this?

**Reply by Chacapamac** [reply]
@aroundi 
**Is this answer your question?**

**Reply by aroundi** [reply]
Hi Chacapamac,

Thank you for getting back to me. I am using another solution for now. But I will set up a public site w/ Astroid and test again and come back with screenshot / video information. I would like to see Astroid keep moving forward. It may take a week or so.

Thanks

**Reply by Chacapamac** [reply]
Happy to help!
